src/pages/iou/request/MoneyTemporaryForRefactorRequestParticipantsSelector.js

```diff
@@ -126,13 +130,51 @@ function MoneyTemporaryForRefactorRequestParticipantsSelector({participants, onF
             [],
             (canUseP2PDistanceRequests || iouRequestType !== CONST.IOU.REQUEST_TYPE.DISTANCE) && ![CONST.IOU.ACTION.CATEGORIZE, CONST.IOU.ACTION.SHARE].includes(action),
             false,
+            false,
+            0,
         );
 
+        return optionList;
+    }, [action, areOptionsInitialized, betas, canUseP2PDistanceRequests, didScreenTransitionEnd, iouRequestType, iouType, options.personalDetails, options.reports, participants]);
+
+    const chatOptions = useMemo(() => {
+        if (!areOptionsInitialized) {
+            return {
+                userToInvite: null,
+                recentReports: [],
+                personalDetails: [],
+                currentUserOption: null,
+                headerMessage: '',
+                categoryOptions: [],
+                tagOptions: [],
+                taxRatesOptions: [],
+            };
+        }
+
+        const newOptions = OptionsListUtils.filterOptions(defaultOptions, debouncedSearchTerm, {
+            betas,
+            selectedOptions: participants,
+            excludeLogins: CONST.EXPENSIFY_EMAILS,
+            maxRecentReportsToShow: CONST.IOU.MAX_RECENT_REPORTS_TO_SHOW,
+        });
+        return newOptions;
+    }, [areOptionsInitialized, betas, defaultOptions, debouncedSearchTerm, participants]);
+    /**
+     * Returns the sections needed for the OptionsSelector
+     * @returns {Array}
+     */
+    const [sections, header] = useMemo(() => {
+        const requestMoneyOptions = chatOptions;
```

Comment:
NAB, Now I think an extra assignment is not needed and we can use `chatOptions` directly.